internal/server/grants_test.go

```diff
@@ -177,7 +178,10 @@ func TestAPI_ListGrants(t *testing.T) {
 						Resource:  "res1",
 					},
 				}
-				assert.DeepEqual(t, grants.Items, expected, cmpAPIGrantShallow)
+				items := slice.Select(grants.Items, func(g api.Grant) bool {
+					return g.Resource != "infra"
+				})
```

Comment:
Why not add to the expected list instead of filtering it out? Seems like this is likely to hide bugs.